std/array.d

```diff
@@ -337,9 +337,10 @@ unittest
 /**
  * Initializes a static array of the given size from the range `r`.
  *
- * If `r` has more than `n` elements, only the first `n` are copied.
- * If `r` has less than `n` elements, the remaining spaces in the static array
- * will have default values.
+ * If `r` has less than `n` elements, only the first `n` are copied, and the
+ * remaining elements are default-initialized.
+ *
+ * A range violation will occur if `r` has more than `n` elements.
```

Comment:
It's kind of the opposite of this I suggested ;-)
I.e. A range violation will occur if `r` has *less* than `n` elements. Having more is fine.